multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/jobs/FilesCleaner.java

```diff
@@ -22,10 +23,12 @@ public class FilesCleaner implements Cleaner {
     private static final Logger LOGGER = LoggerFactory.getLogger(FilesCleaner.class);
 
     private final FileService fileService;
+    private final AsyncUploadJobService uploadJobService;
 
     @Inject
-    public FilesCleaner(FileService fileService) {
+    public FilesCleaner(FileService fileService, AsyncUploadJobService uploadJobService) {
```

Comment:
This cleaner should be split

Reply:
This only deletes the upload job entries. I think it's fine to leave it as is because the entries don't take that much space.
The cleaner that removes files that do not have a started operation isn't written yet.